internal/cli/input/organization.go

```diff
@@ -55,3 +55,20 @@ func GetOrganization(banzaiCli cli.Cli) int32 {
 
 	return id
 }
+
+// GetOrganizations returns a map with the list of organizations
+// where the key is the organization name and the value is the id
+func GetOrganizations(banzaiCli cli.Cli) map[string]int32 {
```

Comment:
```suggestion
func GetOrganizations(banzaiCli cli.Cli) (map[string]int32, err) {
```

don't exit the program if the orgs can't be listed (this function is not interactive in nature)

Reply:
`err` -> `error`